hazelcast/src/main/java/com/hazelcast/config/AbstractXmlConfigBuilder.java

```diff
@@ -92,46 +92,69 @@ public Iterator getPrefixes(String namespaceURI) {
     }
 
     protected void process(Node root) throws Exception {
+        replaceImports(root);
         traverseChildrenAndReplaceVariables(root);
-        replaceImportElementsWithActualFileContents(root);
     }
 
-    private void replaceImportElementsWithActualFileContents(Node root) throws Exception {
+    private void replaceVariables(Node root) throws Exception {
```

Comment:
This and `traverseChildrenAndReplaceVariables(root)` seem to be duplicate methods. Could this be a leftover?

Reply:
Now `traverseChildrenAndReplaceVariables` is an unused method. Please don't forget to remove it in #15824.